src/Conductors/ChecksRole.php

```diff
@@ -47,6 +47,17 @@ public function a($role)
     }
 
     /**
+     * Check if the user doesn't have any of the given roles.
+     *
+     * @param  string  $role
+     * @return bool
+     */
+    public function notA($role)
```

Comment:
Oh, I totally see now. I think it's straightened out in [`b73511b`](https://github.com/JosephSilber/bouncer/commit/b73511bae83a26d4b36b19bf4a06ef5a7c8055e9). Let me know!